WalletWasabi.Fluent/Helpers/CoinHelpers.cs

```diff
@@ -24,7 +24,7 @@ public static LabelsArray GetLabels(this SmartCoin coin, int privateThreshold)
 		return coin.HdPubKey.Cluster.Labels;
 	}
 
-	public static int GetConfirmations(this SmartCoin coin) => coin.Height.Type == HeightType.Chain ? (int)Services.BitcoinStore.SmartHeaderChain.TipHeight - coin.Height.Value + 1 : 0;
+	public static int GetConfirmations(this SmartCoin coin) => coin.Height.Type == HeightType.Chain ? (int)Services.SmartHeaderChain.TipHeight - coin.Height.Value + 1 : 0;
```

Comment:
This whole function feels like a duplicate.

WDYT of this? 

```cs
public static int GetConfirmations(this SmartCoin coin) => coin.Transaction.GetConfirmations((int)Services.BitcoinStore.SmartHeaderChain.ServerTipHeight);
```

As `SmartCoin` has its own `SmartTransaction`, we could use that.

Reply:
Oh shit, I forgot that the `SmartTransaction.GetConfirmations` extension method is not yet on `master`. 😅